tokens/0x2ef27bf41236bd859a95209e17a43fbd26851f92.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
@freeatnet I cant't seem to find where to change the original pull request, so that the smart contract addresses would match. I tried to change it here, to no avail.